### ISSUE TITLE
Ansicon corrupts cmd.exe exit code in Windows Vista x86

### ISSUE THREAD

**Issue by Getfree** [adoxa/ansicon#123]
Steps to reproduce the problem:
1. In a computer with no Ansicon installed, create a file named **test.js** with the following content:
```
SHELL = new ActiveXObject("WScript.Shell") ;
WScript.echo( SHELL.run('cmd /c echo hello', 0, true) ) ;
WScript.echo( SHELL.run('cmd /c non_existing_command', 0, true) ) ;
```
2. Double-click the file to execute it. The result is a number 0 and a number 1.

3. Now, install ansicon and run **test.js** again. The result is a random number twice.

Tested OSs:
Windows XP x86: **pass**
Windows Vista x86: **fail**
Windows 7 x64: **pass**

**Comment by adoxa** [comment]
Identified the problem, but yet to fix (looks like I'll always have to unhook, even if terminating). Is it urgent? I'd rather hold off until I'm ready to make some other changes.

**Comment by Getfree** [comment]
Not really urgent since I can just disable Ansicon when the problem occurs. But it's not ideal, obviously.

**Comment by adoxa** [comment]
Should be fixed in 1.86.

**Comment by Getfree** [comment]
Awesome!
Many thanks.
